setup.py

```diff
@@ -0,0 +1,99 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2018 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import codecs
+import os
+import re
+
+from setuptools import find_packages
+from setuptools import setup
+
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+#####
+# Helper functions
+#####
+def read(*filenames, **kwargs):
+    """
+    Build an absolute path from ``*filenames``, and  return contents of
+    resulting file.  Defaults to UTF-8 encoding.
+    """
+    encoding = kwargs.get('encoding', 'utf-8')
+    sep = kwargs.get('sep', '\n')
+    buf = []
+    for fl in filenames:
+        with codecs.open(os.path.join(HERE, fl), 'rb', encoding) as f:
+            buf.append(f.read())
+    return sep.join(buf)
+
+
+def find_meta(meta):
+    """Extract __*meta*__ from META_FILE."""
+    re_str = r"^__{meta}__ = ['\"]([^'\"]*)['\"]".format(meta=meta)
+    meta_match = re.search(re_str, META_FILE, re.M)
+    if meta_match:
+        return meta_match.group(1)
+    raise RuntimeError('Unable to find __{meta}__ string.'.format(meta=meta))
+
+
+def install_requires():
+    reqs_txt = read('requirements.txt')
+    parsed = reqs_txt.split('\n')
+    parsed = [r for r in parsed]
+    return [r for r in parsed if len(r) > 0]
+
+
+#####
+# Project-specific constants
+#####
+NAME = 'aioshumway'
+PACKAGES = find_packages(where='.')
+META_PATH = os.path.join(NAME, '__init__.py')
+KEYWORDS = ['metrics', 'ffwd', 'heroic', 'asyncio']
+CLASSIFIERS = [
+    'Development Status :: 4 - Beta',
```

Comment:
If we're releasing it as `1.0.0`, I'd argue that it's now `5 - Production/Stable`. However, if you think it's beta, then the version should prob be `0.1.0`.